hack/ci/run-e2e.sh

```diff
@@ -4,7 +4,8 @@ set -o errexit
 
 REPO_ROOT="${REPO_ROOT:-$(dirname "${BASH_SOURCE}")/../..}"
 BINDIR="${BINDIR:-${REPO_ROOT}/bin}"
-IMG="${IMG:-quay.io/jetstack/cert-manager-google-cas-issuer:latest}"
+IMG_REPO="${IMG_REPO:-quay.io/jetstack/cert-manager-google-cas-issuer}"
+IMG_TAG="${IMG_TAG:-latest}"
```

Comment:
Suggestion: set the image tag to something that changes as changes are made to the codebase, i.e a hash of the files. Otherwise I think the cas issuer pod won't get recreated if the script is run a second time against the same test deployment (as there would be no changes to pod spec)